.github/workflows/release.yaml

```diff
@@ -68,3 +181,23 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
 
+  release-cli-via-krew:
```

Comment:
Will this run after `create-release`? Or these two run in parallel?

Reply:
after create-release

Reply:
Yes you are right, I missed `needs: create-release`.